### DISCUSSION TITLE
Icon for multi-instrumentalists

### DISCUSSION THREAD

**Discussion by Petra-Kate** [jamulussoftware/jamulus#2324] [Ideas]
Setting the instrument defines the icon to be displayed in the mixer.
Playing a wind controller, and hence choosing an instrument ad libitum depending on the playing situation, I'd love to see an icon to represent a bunch of different instruments. I'd expect this to be welcomed by "true multi-instrumentalists" as well.

Cheers,
Petra

**Comment by pljones** [comment]
Many "keyboard" players are playing synthesisers - strings, wind, percussion, etc - so there's a need for this from other angles, too.

**Reply by rdica** [reply]
I've often wondered why there isn't an icon for bass+vox, or keys/synths/piano+vox, or most other popular instruments+vox.

**Reply by pljones** [reply]
If people want to provide the icons, which have to conform to the licence, then they can be added fairly easily.
